### ISSUE TITLE
Game Crashes when in the Void

### ISSUE THREAD

**Issue by rickadead** [Cazadorsniper/PO3#596]
Pack Version: 3.0.42
Multiplayer hosted on Server

Description:
Only crashes when going into the void where my Speed farms are. I assume its a block placed there but i couldn't see it listed in the crash report.
As Chunk loading doesn't work with chicken chunks or Claimed areas, nothing is collected when I leave I have to pop back in there and go AFK to get the resources.

Steps to reproduce:
Enter my Void dimension

Crash log
https://pastebin.com/7wtQFdkS

**Comment by Cazadorsniper** [comment]
max-tick-time=-1 in server.properties
Also seems like the issue maybe from the industrial foregoing recollector or something from open computers

**Comment by rickadead** [comment]
Seems like the issue was with a Quantum entangloporter by Mekanism.
Removed it and it hasn't crashed while i was AFK for an hour.
I did try before setting the max tick time to -1 but it didn't stop the crash.

As far as i can see there is nothing from open computers.
Just;
Compact infinite water
wood water tank/irigation/sprinkler
Lamp
seed analyser
Dimensional Transciever
Plant Gather by industrial forgoing (I have these in other dimensions - No crash
)
Imaginary Time block
chunk loader by chicken chunks (although they don't work if your not in the dimension
)


I can only assume it was the Quantum entangloporter and plant gathers causing an issue and removing the Quantum entangloporter seems to have fixed it.

If i find out anything more ill let you know,
Thank you for having a look.
cheers!!
